cms/src/index.ts

```diff
@@ -14,5 +14,34 @@ export default {
 	 * This gives you an opportunity to set up your data model,
 	 * run jobs, or perform some special logic.
 	 */
-	bootstrap(/*{ strapi }*/) {},
+	async bootstrap({ strapi }) {
+		const amountOfUsers = 10;
```

Comment:
we want to have dummy users only in development environment, therefore i suggest to put an if clause with it

`if (process.env.NODE_ENV === 'development') {`